src/main/python/pravega_config.py

```diff
@@ -0,0 +1,101 @@
+"""
+Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
```

Comment:
Recent PRs has already changed the copyrights of the header, please change as well